src/base/net/geoipmanager.cpp

```diff
@@ -451,11 +450,18 @@ void GeoIPManager::downloadFinished(const DownloadResult &result)
                         specialFolderLocation(SpecialFolder::Data) + GEODB_FOLDER);
             if (!QDir(targetPath).exists())
                 QDir().mkpath(targetPath);
-            QFile targetFile(QString::fromLatin1("%1/%2").arg(targetPath, GEODB_FILENAME));
-            if (!targetFile.open(QFile::WriteOnly) || (targetFile.write(data) == -1))
-                LogMsg(tr("Couldn't save downloaded IP geolocation database file."), Log::WARNING);
-            else
+
+            const auto path {QString::fromLatin1("%1/%2").arg(targetPath, GEODB_FILENAME)};
```

Comment:
>And in other similar places across the PR.

In fact, I feel that an excessive abundance of braces worsens the readability of the code, so I would prefer to avoid them in those initializers where it does not bring obvious advantages.

Reply:
>so I would prefer to avoid them in those initializers where it does not bring obvious advantages.

IMO brace initializers is only useful for prohibiting narrowing conversions, other than that it is always assign constructors.

Reply:
It is also useful to avoid confusing notation like `Class object(par1, par2);` that looks like declaration of function `object` that returns result of type `Class`.